packages/edit-site/src/components/template-details/index.js

```diff
@@ -55,13 +56,17 @@ export default function TemplateDetails( { template, onClose } ) {
 	return (
 		<div className="edit-site-template-details">
 			<div className="edit-site-template-details__group">
-				<Heading
-					level={ 4 }
-					weight={ 600 }
-					className="edit-site-template-details__title"
-				>
-					{ title }
-				</Heading>
+				{ template.is_custom ? (
```

Comment:
Should I extract this into the custom util method, similar to `isTemplateRevertable` and `isTemplateRemovable`?